.github/workflows/pull-request.yaml

```diff
@@ -8,7 +8,7 @@ on:
     branches:
       - main
     paths:
-      - .github/workflows/**
+      - .github/workflows/pull-request.yaml
```

Comment:
This was being executed every time we added new workflows. It only makes sense if we only execute the flow if the flow itself has changed.